packages/web/src/config.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  Instrumentation,
+  MetaItem,
+  App,
+  Session,
+  User,
+  Config,
+  defaultGlobalObjectKey,
+  Transport,
+} from '@grafana/agent-core';
+
+import { ErrorsInstrumentation, WebVitalsInstrumentation } from './instrumentations';
+import { browserMeta, pageMeta } from './metas';
+import { FetchTransport } from './transports';
+
+export interface BrowserConfig {
+  url?: string;
+  apiKey?: string;
+  app: App;
+  session?: Session;
+  user?: User;
+  globalObjectKey?: string;
+  preventGlobalExposure?: boolean;
+  metas?: MetaItem[];
+  instrumentations?: Instrumentation[];
+  transports?: Transport[];
+}
+
+export const defaultMetas: MetaItem[] = [browserMeta, pageMeta];
+
+export const getDefaultInstrumentations = (): Instrumentation[] => [
```

Comment:
Should we add a configuration parameter here to include tracing and console instrumentation as well? This will prevent some additional manual package installations.

Reply:
`getDefaultInstrumentations` -> `getWebInstrumentations`?

Reply:
sure, but still can't add tracing without balooning the bundle

Reply:
Yup, that's true. Let's keep tracing out of this.

Reply:
done